clarifai/models/model_serving/docs/inference_parameters.md

```diff
@@ -0,0 +1,82 @@
+## Inference paramters
+
+When making prediction, you may need to change some paramters to adjust the result. Those paramters will be passed through `paramters()` of a request in triton python model.
+
+In order to send it to `**kwargs` of `get_predictions` in `inference.py`, there are 2 ways:
+1. You can send any arbitrary parameters via clarifai API.
```

Comment:
We're removing this, they will need to be defined by the model owner

Reply:
Sure, error will arise if user uses undefined parameters in test cases